src/redux/actions/login-logout.js

```diff
@@ -0,0 +1,5 @@
+import { LOGIN, LOGOUT } from '../../scripts/constants';
```

Comment:
I think we should not contain actionTypes in scripts directory it should be in redux directory